doc/development.md

```diff
@@ -97,6 +97,8 @@ Config: `string`
 
 The test should be added to the init function inside of the test file. If the test module is being created then an `init` function should be created which registers the tests and the package must be imported inside of `tests/registry/registry.go` to allow for discovery.
 
+A GUID may be required in the following fields of a Test object: In, Out, and Config. Replace all GUIDs with GUID varaibles which take on the format `$uuid<num>` (e.g. $uuid123). Where `<num>` must be a positive number. GUID variables with identical `<num>` fields will be replaced with identical GUIDs. As an example, look at the following file: tests/positive/partitions/zeros.go.
```

Comment:
s/A GUID/UUIDs

Reply:
s/positive number/positive integer

Reply:
s/As an example.../For example, look at [test/.../zeroes.go](link to file on github)